docs/driver-only-builds.md

```diff
@@ -243,6 +243,29 @@ The same holds for the associated algorithm:
 `[PSA_WANT|MBEDTLS_PSA_ACCEL]_ALG_FFDH` allow builds accelerating FFDH and
 removing builtin support (i.e. `MBEDTLS_DHM_C`).
 
+RSA
+---
+
+It is possible for all RSA operations to be provided only by a driver.
+
+More precisely, if:
+- all the RSA algorithms that are enabled (`PSA_WANT_ALG_RSA_*`) are also
+  accelerated (`MBEDTLS_PSA_ACCEL_ALG_RSA_*`),
+- and all the RSA key types that are enabled (`PSA_WANT_KEY_TYPE_RSA_*`) are
+  also accelerated (`MBEDTLS_PSA_ACCEL_KEY_TYPE_RSA_*`),
+
```

Comment:
Not sure why the new line is here?

Reply:
IIRC (github) markdown wants a new line at the end of a list (as opposed to the beginning where the absence of a new line is now accepted - IIRC original markdown wanted new lines on both ends of a list).

Let's try it:
- this list
- is not followed by a new line.
This should not be part of the list.

Let's try it:
- this list
- is followed by a new line.

This should not be part of the list.

Looks like I remembered correctly then (at least for github markdown, I'm not sure how unified the markdown ecosystem is these days).

Reply:
[CommonMark](https://spec.commonmark.org/0.30/#example-302) doesn't require a blank line above lists, but historical markdown did, and `pandoc` does by default. We render some (all now?) markdown document on readthedocs, so we should keep our documents portable there. I don't know how it handles this specific case.

Reply:
In doubt, I went ahead and added new lines before each list. I think for people who read the unrendered text, this also makes the new line after the list more natural.